projects/samples/robotbenchmark/highway_driving/worlds/.highway_driving.wbproj

```diff
@@ -8,5 +8,5 @@ selectionDisabled: 0
 viewpointLocked: 0
 orthographicViewHeight: 1
 textFiles: 0 "controllers/highway_driving/highway_driving.py"
-robotWindow: supervisor
+consoles: Console:All:All
```

Comment:
The robot window's node name (and type) changed so it was not found, but it still exists:
```suggestion
robotWindow: robot
consoles: Console:All:All
```